test/sql/bind.test.lua

```diff
@@ -107,7 +107,7 @@ test_run:cmd("setopt delimiter ';'")
 if remote then
 	execute("CREATE TABLE t(a VARBINARY PRIMARY KEY);")
 	execute("INSERT INTO t VALUES (X'00');")
-	res = execute("SELECT typeof(?);", box.space.T:select()[1])
+    res = execute("SELECT TYPEOF(?);", box.space.t:select()[1])
```

Comment:
Typo: misindent.

Reply:
I used 4 spaces instead of a tab, so it visually looks this way.